command.c

```diff
@@ -251,6 +268,23 @@ static int redis_argkvx(struct cmd *r) {
     return 0;
 }
 
+/*
+ * Check if command type expects a sub-command before the key
+ * Format: COMMAND SUBCOMMAND key [...]
```

Comment:
```suggestion
 * Format: command subcommand key [ arg ... ]
```

Reply:
Thanks, changed tat the same time as the others